frontend/simple/views/UserProfile.vue

```diff
@@ -208,7 +211,13 @@
           </div>
         </div>
         <div class="has-text-centered button-box">
-          <button class="button is-success is-large" :disabled="$v.editedGroupProfile.$invalid" v-if="currentGroupContractId"><i18n>Save Group Profile</i18n></button>
+          <button
+            class="button is-success is-large"
+            :disabled="$v.editedGroupProfile.$invalid"
+            v-if="currentGroupContractId"
+            data-test="GroupProfileSubmitBtn">
```

Comment:
I believe we don't need 'GroupProfile' prefix in all `data-test`... do we?

Reply:
I've added them to distinguish from the standard `profile...` prefixed `data-test`s on the same page.

Reply:
okidoki then :)